src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -255,52 +254,53 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
             }),
         );
       }
-      const data = await Promise.all(promises);
+      const getData = await Promise.all(promises);
```

Comment:
suggest renaming `getData` to `results`. Our naming convention starting with "get" usually indicates it is a method.

Reply:
That makes sense, made the change.